src/actions/vstudio/vs2010.lua

```diff
@@ -31,7 +31,10 @@
 		["file.basename"]               = { absolute = false, token = "%(Filename)" },
 		["file.abspath"]                = { absolute = true,  token = "%(FullPath)" },
 		["file.relpath"]                = { absolute = false, token = "%(Identity)" },
-		["file.path"]                   = { absolute = true,  token = "%(Identity)" },
+		["file.path"]                   = { absolute = false, token = "%(Identity)" },
```

Comment:
Yes, %(Identity) is a relative path in msbuild... and in premake itself, path is an alias for relpath.